app/models/warehouse/analytics/referral_contact.rb

```diff
@@ -19,7 +19,6 @@ def self.sync!
         batch = []
         match_contacts.each do |match_contact|
           batch << new(
-            id: match_contact.id,
```

Comment:
Not sure why we had this specified, but we can't rely on it now that we're faking contacts.